test/pyTest/testLink.py

```diff
@@ -0,0 +1,50 @@
+import time
+import tempfile
+import os
+import shutil
+
+from pySequenceParser import sequenceParser as seq
+from . import createFile, createFolder, getSequencesFromPath
+
+from nose.tools import *
+
+root_path = ''
+
+def setUp():
+    global root_path
+    root_path = tempfile.mkdtemp()
+
+    # create files
+    files_to_create = [
+        "plop.txt"
+    ]
+    for f in files_to_create:
+        createFile(root_path, f)
+
+    # create folder
+    createFolder(root_path, "dir1")
+
+    # create sym link
+    os.symlink(os.path.join(root_path, "plop.txt"), os.path.join(root_path, "plop_sym_link.txt"))
+    os.symlink(os.path.join(root_path, "dir1"), os.path.join(root_path, "dir2"))
+
+def tearDown():
+    global root_path
+    shutil.rmtree(root_path)
+
+def testLinkType():
+    """
+    Check stats of a synbolic link folder.
```

Comment:
Could you update this doc? Something like:
"""
Check the type of the items found.
"""